### PR TITLE
Display screen session if you set theme_display_screen to yes

### PR DESCRIPTION
**Pull request by pfsmorigo**
I use multiple screen sessions so I would like to see which one I am in the current terminal.

### CODE REVIEW

File: functions/__bobthefish_colors.fish [hunk 1]
```diff
@@ -29,6 +29,7 @@ function __bobthefish_colors -S -a color_scheme -d 'Define colors used by bobthe
       set -x color_k8s                      magenta white --bold
       set -x color_username                 white black --bold
       set -x color_hostname                 white black
+      set -x color_screen                   brgreen $colorfg
```
Comment (bobthecow) [inline functions/__bobthefish_colors.fish:32]: can you add an example segment to `bobthefish_display_colors`, then use that to upload screenshots of how it looks in all the themes?

File: fish_prompt.fish [hunk 2]
```diff
@@ -911,6 +912,18 @@ function __bobthefish_prompt_hg -S -a hg_root_dir -a real_pwd -d 'Display the ac
...
 
+function __bobthefish_prompt_screen -S -d 'Display the screen name'
+    [ "$theme_display_screen" != 'yes' ]
+    and return
+
+    [ -z "$STY" ]
+    and return
+
+    __bobthefish_start_segment $color_screen
+    echo -ns (command echo $STY | cut -d. -f2) ' '
```
Comment (bobthecow) [inline fish_prompt.fish:923]: we can probably do this entirely with the `string` builtin, saving ourselves a couple of calls out to core utils. what does $STY typically contain?

**Review by bobthecow — CHANGES_REQUESTED** [review]
Awesome. I like the idea, I like making it opt-in rather than opt-out.

Is there an analog the other settings which allow a verbose version, where it'd be useful to know you're in a screen session, but not necessarily which one? I imagine for someone who typically only ran one screen session per machine, that would be preferable.

File: fish_prompt.fish (hunk 2 above)
Comment (pfsmorigo) [reply fish_prompt.fish]: done

**Comment by pfsmorigo** [comment]
> Awesome. I like the idea, I like making it opt-in rather than opt-out.
> 
> Is there an analog the other settings which allow a verbose version, where it'd be useful to know you're in a screen session, but not necessarily which one? I imagine for someone who typically only ran one screen session per machine, that would be preferable.

Now is opt-in without verbose. I don't know what to show if verbose is off. Now only adds a space with the color. Maybe we can give me an idea.

About the screen name, the format is pid.tty.host: 

```
> screen -list
There are screens on:
        3226255.first  (09/03/2020 08:40:23 AM)        (Attached)
        2824062.second (09/03/2020 05:48:22 AM)        (Attached)
        1901576.third  (08/30/2020 09:18:26 AM)        (Attached)
3 Sockets in /run/screen/S-pfsmorigo.
> echo $STY
3226255.second
```
So I just use the second part (tty) as the screen session.

**Comment by pfsmorigo** [comment]
I don't know if I need to do something else. I think I did all requested changes.

**Comment by pfsmorigo** [comment]
@bobthecow Hello. I think I did all the requested modifications. What do I need to do in order to have this merged? Thanks!

**Comment by pfsmorigo** [comment]
@bobthecow Hello again! Is my pull request ok or should I change something?

**Comment by pfsmorigo** [comment]
Hello, is there something missing from my side? Can we merge this?

**Comment by pfsmorigo** [comment]
@bobthecow hello, can you merge this PR? Please let me know if you want me to change anything.

**Comment by bobthecow** [comment]
@pfsmorigo hi! I'm so sorry, for some reason I missed the pings entirely. I'll take a look!

**Review by bobthecow — APPROVED** [review]

**Comment by bobthecow** [comment]
I don't think I have a great thing to do when not verbose, so I'll merge this then remove that option ¯\\\_(ツ)\_/¯

**Comment by pfsmorigo** [comment]
Thank!

**Pull request merged**